pkg/operator/k8sutil/pod.go

```diff
@@ -89,36 +86,11 @@ func MakeRookImage(version string) string {
 	return fmt.Sprintf("%s/rook:%v", repoPrefix(), getVersion(version))
 }
 
-func PodWithAntiAffinity(pod *v1.Pod, attribute, value string) {
```

Comment:
Not used anywhere and not needed since this way of providing affinity is ALPHA and has been replaces in kubernetes